truenas/webui
# NAS-112109 / 21.10 / NAS-112109: Handling broken reporting database

How to test?

1. Open `System Settings -> Shell`
2. Update system time to future, type `date --set='+14 day +1 hour'` for example
3. Reboot the system, type `reboot`
4. Wait until the system is loaded
5. Open Dashboard and check the Network widget
6. Fix reporting database

Repeat steps 1-4
1. Go to Reporting 
2. Fix reporting database
